packages/kit/src/exports/index.js

```diff
@@ -35,8 +35,8 @@ export function error(status, body) {
 }
 
 /**
- * Create a `Redirect` object. If thrown during request handling, SvelteKit will return a redirect response.
- * Make sure you're not catching the thrown redirect, which would prevent SvelteKit from handling it.
+ * Crée un objet `Redirect`. Si levé pendant le traitement d'une requête, Sveltekit va retourner une réponse de redirection.
```

Comment:
```suggestion
 * Crée une exception de type `Redirect`. Si levé pendant le traitement d'une requête, Sveltekit va retourner une réponse de redirection.
```

avec "thrown", il y a plus la notion d'exception (quand même masqué en anglais) que j'ajouterais en français

Reply:
Hmm je ne sais pas. "Créer une exception" c'est ambigu, on peut penser qu'on va la lever, alors que la fonction `redirect` ne `throw` pas, il faut `throw` manuellement après.

Reply:
Mais Redirect n'est pas un objet...

Reply:
Si, justement https://kit.svelte.dev/docs/types#public-types-redirect